runtime/runtime/src/verifier.rs

```diff
@@ -796,6 +796,31 @@ mod tests {
         }
     }
 
+    pub fn verify_validate_and_charge_transaction(
```

Comment:
```suggestion
    pub fn validate_verify_and_charge_transaction(
```

As the functions are correspondingly `validate_transaction` and `verify_and_charge_transaction`.